chainguard-dev/edu
# adding Images Directory walkthrough

## Type of change

This PR adds a doc outlining how to use the Images Directory, with a special focus on highlighting the SBOM and Vulns tabs.

It also adds a brief section to our existing doc on retrieving an Image's SBOM that highlights the SBOM tab.

### What should this PR do?

Resolves https://github.com/chainguard-dev/internal/issues/3550

### Why are we making this change?


### What are the acceptance criteria? 




### How should this PR be tested?

Confirm that my instructions and explanations make sense. Also, please (as always) give a close eye to formatting and style.
